dts/bindings/gpio/st,stm32-gpio.yaml

```diff
@@ -17,6 +21,10 @@ properties:
   "#gpio-cells":
     const: 2
 
+  ngpios:
+    type: int
+    default: 16
```

Comment:
The rationale behind `default` values must _always_ be documented in the binding.